frontend/src/Header.js

```diff
@@ -1,16 +1,28 @@
 import React from 'react'
 import styled from 'styled-components/macro'
+import Logo from './images/Logo.png'
```

Comment:
logo würde ich in dem Fall klein schreiben, weil es keine component ist und dann unten bei source auch als logo kleingeschrieben importieren.

Reply:
danke für den Hinweis.